tpm2_constants.h

```diff
@@ -9,57 +9,62 @@
  *
  */
 
+
+#ifndef _AT
```

Comment:
Why is this before #ifndef _TPM2_CONSTANTS_H?

Reply:
I was being sloppy ( ._.)